Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -831,12 +865,12 @@ def test_get_all_activities___Guest(self):
         assert response.json()["ActivityCode"] == "360"
         assert response.json()["ActivityDescription"] == "360.gordon.edu"
         assert response.json()["ActivityImagePath"] == "https://360apitrain.gordon.edu/browseable/uploads/360/canvasImage.png"
-        assert response.json()["ActivityBlurb"] == ""
+        assert response.json()["ActivityBlurb"] == "This is me changing the description"
         assert response.json()["ActivityURL"] == "http://360.gordon.edu"
         assert response.json()["ActivityType"] == "STU"
         assert response.json()["ActivityTypeDescription"] == "Student Life"
         assert response.json()["Privacy"] == False
-        assert response.json()["ActivityJoinInfo"] == ""
+        assert response.json()["ActivityJoinInfo"] == "me adding special information"
         
 
 #    Verify that an activity leader can a single activity.
```

Comment:
Maybe this comment should say something like "verify leader can get all info for an activity"???